api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/request/DocsRequestPath.java

```diff
@@ -0,0 +1,108 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi.request;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
+
+import javax.servlet.http.HttpServletRequest;
+
+import edu.cornell.mannlib.vitro.webapp.dynapi.request.ApiRequestPath.RequestType;
+
+public class DocsRequestPath {
```

Comment:
Consider introducing an interface or abstract root for DocsRequestPath and ApiRequestPath?

Reply:
Deferring for future work.